src/node_osrm.cpp

```diff
@@ -573,6 +575,48 @@ void Engine::nearest(const Nan::FunctionCallbackInfo<v8::Value> &args)
     Run(args, std::move(params));
 }
 
+void Engine::tile(const Nan::FunctionCallbackInfo<v8::Value> &args)
+{
+    Nan::HandleScope scope;
+    if (args.Length() < 2)
+    {
+        Nan::ThrowTypeError("two arguments required");
+        return;
+    }
+
+    v8::Local<v8::Object> obj = Nan::To<v8::Object>(args[0]).ToLocalChecked();
+
+    v8::Local<v8::Value> x = obj->Get(Nan::New("x").ToLocalChecked());
+    if (!x->IsNumber() && !x->IsUndefined())
```

Comment:
Related to https://github.com/Project-OSRM/osrm-backend/issues/2009: this does not catch nans and infs, does it?

Should we handle them on the API level?